src/controllers/legacycontrollermapping.h

```diff
@@ -32,6 +32,12 @@ class LegacyControllerMapping {
         bool builtin;
     };
 
+    enum DeviceDirection {
+        OUT = 0x1,
```

Comment:
such issuers normally happen a ; or } is missing in the code before probably in one of the included headers.